<!-- Launch file for Ground Filter -->
<launch>
	<arg name="subscribe_topic" default="/velodyne_points" />
	<arg name="remove_ground" default="true" />
	<arg name="points_distance" default="0.1" />
	<arg name="angle_threshold" default="0.35" />
	
	<!-- rosrun lidar_tracker ground_filter -->
	<node pkg="ground_filter" type="ground_filter_node" name="ground_filter_node">
		<param name="subscribe_topic" value="$(arg subscribe_topic)" />
		<param name="remove_ground" value="$(arg remove_ground)" />
		<param name="points_distance" value="$(arg points_distance)" />
		<param name="angle_threshold" value="$(arg angle_threshold)" />
	</node>

    <arg name="bag" default="rosbag" />
    <arg name="bag_path" default="$(find ground_filter)/$(arg bag).bag" />
    <node name="playbag" pkg="rosbag" type="play" args="--clock $(arg bag_path)" required="false" />
</launch>
